refinery/core/models.py

```diff
@@ -95,13 +95,11 @@ class UserProfile(models.Model):
     catch_all_project = models.ForeignKey('Project', blank=True, null=True)
     login_count = models.IntegerField(default=0)
 
-    def __unicode__(self):
-        return (
-            str(self.user.first_name) + " " +
-            str(self.user.last_name) + " (" +
-            str(self.affiliation) + "): " +
-            str(self.user.email)
-        )
+    def __str__(self):
```

Comment:
`python_2_unicode_compatible()` decorator should be added:
https://docs.djangoproject.com/en/1.8/topics/python3/#str-and-unicode-methods

Reply:
Yep, "Django provides a simple way to define __str__() and ` __unicode__()`_ methods that work on Python 2 and 3: you must define a __str__() method returning text and to apply the python_2_unicode_compatible() decorator."

Reply:
Right, so where's the decorator? :)

Reply:
@hackdna For future reference, I'll tag or msg you when a pull request is ready for another round of review.